.openpublishing.redirection.json

```diff
@@ -412,273 +412,273 @@
             "redirect_document_id": true
         },
         {
             "source_path": "docs/standard/exceptions/exception-handling-fundamentals.md",
-            "redirect_url": "/dotnet/articles/standard/exceptions/"
+            "redirect_url": "/dotnet/standard/exceptions/"
         },
         {
             "source_path": "docs/standard/exceptions/exception-hierarchy.md",
-            "redirect_url": "/dotnet/articles/standard/exceptions/"
+            "redirect_url": "/dotnet/standard/exceptions/"
         },
         {
             "source_path": "docs/standard/garbagecollection/fundamentals.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/fundamentals"
+            "redirect_url": "/dotnet/standard/garbage-collection/fundamentals"
         },
         {
             "source_path": "docs/standard/garbagecollection/gc.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/gc"
+            "redirect_url": "/dotnet/standard/garbage-collection/gc"
         },
         {
             "source_path": "docs/standard/garbagecollection/implementing-dispose.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/implementing-dispose"
+            "redirect_url": "/dotnet/standard/garbage-collection/implementing-dispose"
         },
         {
             "source_path": "docs/standard/garbagecollection/index.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/"
+            "redirect_url": "/dotnet/standard/garbage-collection/"
         },
         {
             "source_path": "docs/standard/garbagecollection/induced.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/induced"
+            "redirect_url": "/dotnet/standard/garbage-collection/induced"
         },
         {
             "source_path": "docs/standard/garbagecollection/latency.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/latency"
+            "redirect_url": "/dotnet/standard/garbage-collection/latency"
         },
         {
             "source_path": "docs/standard/garbagecollection/unmanaged.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/unmanaged"
+            "redirect_url": "/dotnet/standard/garbage-collection/unmanaged"
         },
         {
             "source_path": "docs/standard/garbagecollection/using-objects.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/using-objects"
+            "redirect_url": "/dotnet/standard/garbage-collection/using-objects"
         },
         {
             "source_path": "docs/standard/garbagecollection/weak-references.md",
-            "redirect_url": "/dotnet/articles/standard/garbage-collection/weak-references"
+            "redirect_url": "/dotnet/standard/garbage-collection/weak-references"
         },
         {
             "source_path": "docs/standard/getting-started.md",
```

Comment:
I think some weird things happened when merging changes from master or maybe something I've overlooked. I think there's no /articles left in this file.